packages/scope-manager/src/referencer/PatternVisitor.ts

```diff
@@ -1,5 +1,5 @@
 import { TSESTree, AST_NODE_TYPES } from '@typescript-eslint/types';
-import { VisitorBase, VisitorOptions } from './Visitor';
+import { VisitorBase, VisitorOptions } from './VisitorBase';
```

Comment:
this change should be ported to master, we have circular dependencies because of that